src/LogicApp/LogicApp/Cmdlets/IntegrationAccount/GetAzureIntegrationAccountAssemblyCommand.cs

```diff
@@ -0,0 +1,91 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+namespace Microsoft.Azure.Commands.LogicApp.Cmdlets
+{
+    using Microsoft.Azure.Commands.LogicApp.Utilities;
+    using Microsoft.Azure.Commands.ResourceManager.Common;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+    using Microsoft.Azure.Management.Logic.Models;
+    using System.Management.Automation;
+
+    /// <summary>
+    /// Gets the integration account assembly
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, AzureRMConstants.AzureRMPrefix + "IntegrationAccountAssembly", DefaultParameterSetName = ParameterSet.ByIntegrationAccount)]
+    [OutputType(typeof(AssemblyDefinition))]
```

Comment:
All output types and input parameter types that come from the management client should be wrapped in a "PS\<ModelName\>" class.  This allows you to mitigate breaking changes such as properties being removed or renamed in the SDK model by simply adding them back to the PS model.  This would allow you to bump SDK versions with breaking changes without waiting for our breaking change release.

Reply:
Makes sense. Added